mono/metadata/object.c

```diff
@@ -6316,19 +6317,19 @@ mono_string_new_checked (MonoDomain *domain, const char *text, MonoError *error)
 	MonoString *o = NULL;
 	guint16 *ut;
 	glong items_written;
-	int l;
+	int len;
 
+	g_assert (error);
```

Comment:
This is not useful, since we are going towards an initialization where we allocate them and at API boundaries.